api/resources_portal/test/views/test_organization.py

```diff
@@ -49,46 +55,83 @@ def setUp(self):
         self.url = reverse("organization-detail", args=[self.organization.id])
 
     def test_get_request_returns_a_given_organization(self):
```

Comment:
Do we want a test case for unauthenticated (anonymous) users failing? To login as anonymous user, use `self.client.force_authenticate(user=None)`

Reply:
How is this different than just not calling `self.client.force_authenticate` at all? Aren't the requests by default made by a unauthenticated user?

Reply:
That's probably true. Still, might be good to have an unauthenticated test case.

Reply:
Done.